### PR TITLE
Open privacy dialog directly from datasets/maps items

### LINKED ISSUE #2061: Open privacy dialog directly from an item
Since there is a private indicator directly on the items, e.g. maps:
screen shot 2015-02-04 at 09 38 03

datasets:
image

why not simply make it clickable to open the dialog directly, instead of having to select an item and go through the filters menu?

### LINKED ISSUE #2430: Change Privacy - Copy Edit: “protected” should be “protect”

### LINKED ISSUE #2432: Bigger Size → “Largest size” probably better grammatically

### PR DESCRIPTION
**Pull request by viddo**
Fixes #2061  - 
privacy-dialog

Implemented by extracting the dialog creation from the filters view, and using the `cdb.god` model as a the global event bus (added documentation for it). Also extracted/simplified the code by extracting the `dialogOpened`/`dialogClosed` events to the base dialog instead, to not have to bind/re-trigger events through a tree of views.

The privacy dialog will only open if you're the owner of the item.

---

Fixes #2430  - screen shot 2015-02-23 at 15 24 18

--- 

Fixes #2432  - screen shot 2015-02-23 at 16 08 42

### CODE REVIEW

**Comment by viddo** [comment]
cc @CartoDB/frontend code review please :)

**Comment by viddo** [comment]
cc @saleiva for the visual fixes that I piggyback in this PR

**Comment by saleiva** [comment]
:+1:

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1836/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1867/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1868/))

**Comment by viddo** [comment]
bump, still waiting for the code review to merge.. ;)

**Comment by xavijam** [comment]
oh... reviewing

**Comment by xavijam** [comment]
By the way, I should use god thing for create-dialog and not the binding+triggering events through a tree of views :(

File: lib/assets/javascripts/cartodb/new_dashboard/background_importer/background_importer_item_view.js [hunk 1]
```diff
@@ -104,15 +103,10 @@ module.exports = cdb.core.View.extend({
   },
 
   _showImportStats: function() {
-    var self = this;
-    this.trigger('dialogOpened', this);
-
     (new SuccessDetailsDialog({
+      clean_on_hide: true,
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/background_importer/background_importer_item_view.js]: ha :D

**Comment by xavijam** [comment]
Go ahead and fly!

**Pull request merged**